src/main/java/org/terasology/tutorialWorldGenerationTrees/TreesRasterizer.java

```diff
@@ -70,7 +73,7 @@ public void generateChunk(CoreChunk chunk, Region chunkRegion) {
             Vector3i treeMinimumPos = new Vector3i(treePosition).sub(radius, 0, radius);
 
             // creates regions for different parts of a tree
-            Region3i treeRegion = Region3i.createFromMinAndSize(treeMinimumPos, new Vector3i(width, height, width));
+            Region3i treeRegion = BlockRe.createFromMinAndSize(treeMinimumPos, new Vector3i(width, height, width));
```

Comment:
To avoid such accidents, make sure to at least compile before pushing code - ideally before declaring a PR ready for review you should compile the code, run the tests of the affected module (if any) and start a gameplay (e.g. `CoreSampleGameplay`) with the affected module being activated (e.g. via Advanced Game Setup (AGS)).